### ISSUE TITLE
Remove pending tests

### ISSUE THREAD

**Issue by nzakas** [eslint/eslint#5126]
I've recently noticed that when our tests run, there are three pending tests. Pending tests look like this:

```js
it("some description")
```

I haven't been able to track down where these are, but we should remove them.

**Comment by gyandeeps** [comment]
They were introduced here: https://github.com/eslint/eslint/pull/4467
I am not sure what the reson was. Maybe we want to activate those tests?

**Comment by platinumazure** [comment]
I looked at [lo1tuma/eslint-plugin-mocha](https://github.com/lo1tuma/eslint-plugin-mocha) and saw there was unfortunately no rule for this scenario. But I wanted to ask anyway, is it worth opening an issue in that project and then consuming eslint-plugin-mocha as a devDependency to avoid this in the future?

**Comment by IanVS** [comment]
I added them in.  They are for functionality I planned to add to autoconfig, but didn't yet have a chance to.    They won't pass yet, so they shouldn't be enabled.  I'll submit a PR to remove them, then re-add them when I have a chance to implement the enhancements to autoconfig.

**Comment by IanVS** [comment]
@nzakas, these tests actually of the form:

```js
xit("test description", function() {
    // actual test with assertions
});
```

**Comment by IanVS** [comment]
@platinumazure There is https://github.com/Nate-Wilkins/eslint-plugin-bdd/blob/master/docs/rules/exclude.md, which will check for `xit` and `xdescribe`.

**Comment by platinumazure** [comment]
@IanVS :+1: I didn't spot that plugin.
